lib/recurly/xml.rb

```diff
@@ -89,9 +89,9 @@ def to_s
   insecure_noko_msg = <<-MSG
 
     You are attempting to use an insecure version of
-    nokogiri on an insecure version of ruby. Please see
+    nokogiri or an insecure version of ruby. Please see
```

Comment:
Should we just consider showing a completely different error for nokogiri vs ruby? Maybe it doesn't matter that much, but at the same time, we're checking for all these different scenarios, we might as well tell the user which one needs to be upgraded to pass our checks.